jane/doc/extensions/_06-uniqueness/pitfalls.md

```diff
@@ -28,7 +28,7 @@ lift out the allocation so that `mk ()` returns the same pointer on each
 invocation. However, you might want to annotate this allocation as `unique`:
```

Comment:
```suggestion
invocation. However, you might want to annotate this allocation's type as `unique`:
```

Reply:
Done (in #3955). Is there more to do in this Discussion?

Reply:
I think this one can be closed